artemis-cli/src/main/java/org/apache/activemq/artemis/integration/bootstrap/ActiveMQBootstrapLogger.java

```diff
@@ -23,23 +23,14 @@
 /**
  * Logger Code 10
  */
-@LogBundle(projectCode = "AMQ", regexID = "10[0-9]{4}")
+@LogBundle(projectCode = "AMQ", regexID = "10[0-9]{4}", retiredIDs = "101001, 101002, 101003")
```

Comment:
This list isn't necessarily meant to be exhaustive at the moment. I just took the IDs from the `@LogMessage` instances that I removed. We may have duplicates eventually from something we've removed historically, but this should give us a sensible way to avoid duplicates moving forward. In the future it might be worth adding IDs that were removed historically.

Reply:
For anyone following up later, more IDs were retired/excluded later via PR #5320.